site/app/models/Email.php

```diff
@@ -25,10 +23,6 @@ class Email extends AbstractModel {
     /** @property @var string user name */
     protected $user_id;
 
-    /** @property @var string Intended receiver of email */
-    // NOTE: THIS IS ESSENTIALLY A DEPRECATED / LEGACY FIELD
-    protected $recipient;
```

Comment:
Leave this in for now. Removing this should be accompanied with a migration to delete the column from the DB.